app/lint.xml

```diff
@@ -3,6 +3,7 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <lint>
+    <issue id="ButtonStyleXmlDetector" severity="ignore" />
```

Comment:
I believe this is a detector we wrote – why do we want to ignore it?

(I wonder if we should name these e.g. `MozillaButtonStyle...` so that it's clear we wrote these)

Reply:
Oh, yeah, I didn't know that. I was expecting this to be something new. But looking at your PR it seems like we have to fix the lint check.